Automattic/wp-calypso
# Add Jetpack manual activation step with license key clipboard

#### Changes proposed in this Pull Request

* Adds the last step of the Jetpack manual activation flow which shows users their license key.

#### Testing instructions

* Download this PR.
* Start Calypso with `yarn start`. 
* Go to `http://calypso.localhost:3000/checkout/jetpack/thank-you/licensing-manual-activate/jetpack_backup_daily` (you can try with other product slugs).
* Continue the flow until you arrive at `http://calypso.localhost:3000/checkout/jetpack/thank-you/licensing-manual-activate-license-key/:product`.
* Verify the product mentioned in the title matches the one in the URL.
* Verify you can copy the license key by clicking on the `Copy` button.
* Verify the UI looks fine in different viewport sizes.

**Notes**
- The license key is hardcoded. We still need to wire up some parts of the flow to carry the license key from the start to this step.
- I added support for a loading state in the `<LicensingActivating />` component that can be activated by passing the `isLoading` boolean prop.

Related to 1201096622142517-as-1201096622142548

#### Demo

##### Loading state
![image](https://user-images.githubusercontent.com/3418513/140524164-35ec3e03-42c3-43b9-bfca-5aa302c951d8.png)

##### Loading finished
![image](https://user-images.githubusercontent.com/3418513/140523941-3a904240-e039-4c2d-a60d-38d11c2cdde5.png)
